gpii/node_modules/flowManager/test/shared/BrowserChannelTestDefs.js

```diff
@@ -53,6 +53,12 @@ gpii.tests.flowManager.browserChannel.reportPlatform = function () {
     };
 };
 
+// A no-op to gracefully fail fetching the solutions registry from the source
+// code repository in order to use the canned registry set up for these tests.
+gpii.tests.flowManager.browserChannel.getRepositorySolutions = function (/*repositorySolutionsLoader*/) {
```

Comment:
Please double check if this function is still needed. Thanks.

Reply:
Since it was removed from the config above, it can be removed here too.